Tools/Taxonomy/Program.cs

```diff
@@ -17,7 +18,7 @@ internal class Program
         private static void Main(string[] args)
         {
             // see README.MD for usages
-            bool result = Parser.Default.ParseArguments<AddOwaspRelationshipToCweOptions, CweOptions, NistOptions, OwaspOptions, PciOptions, WascOptions>(args)
+            bool result = Parser.Default.ParseArguments<AddOwaspRelationshipToCweOptions, CweOptions, DisaOptions, NistOptions, OwaspOptions, PciOptions, WascOptions>(args)
```

Comment:
>AddOwaspRelationshipToCweOptions, CweOptions, DisaOptions, NistOptions, OwaspOptions, PciOptions, WascOptions [](http://example.com/codeflow?start=56&length=109)

let's break the options, one by line.
the string is getting too long

Reply:
sounds good, updated.